Add documentation about using custom OpenAI models

tm-kn opened the pull request.

tm-kn commented on docs/ai-backends.md:
```diff
@@ -107,3 +107,46 @@ WAGTAIL_AI = {
+
+llm supports adding custom OpenAI models. This may be necessary if:
+
+- You want to use a model that's not supported by llm package yet.
```
I do wonder if we want to monospace the library name. We don't write `Wagtail`, `Windows`, `Django` etc.

zerolab replied:
I don't mind either way, as long as we are consistent throughout

tm-kn commented:
@zerolab Thanks for your help. I've updated the documentation now and made it more consistent. Let me know how you feel about it.

The "LLM" library's name is all uppercase (https://llm.datasette.io/en/stable/) so I've just started using that instead and I've added the double quotes to make sure people don't think it means the acronym for large language models.

zerolab commented on docs/ai-backends.md:
```diff
@@ -107,3 +112,47 @@ WAGTAIL_AI = {
+
+You can find the "LLM" library specific instructions at: https://llm.datasette.io/en/stable/other-models.html#adding-more-openai-models.
+
+1. Find the "LLM" library's directory. You can set a custom one with the. You can
+   set a custom one with the
```
```suggestion
1. Find the "LLM" library's directory. You can set a custom one with the
```

zerolab commented on docs/ai-backends.md:
```diff
@@ -107,3 +112,47 @@ WAGTAIL_AI = {
+1. Find the "LLM" library's directory. You can set a custom one with the. You can
+   set a custom one with the
+   [`LLM_USER_PATH`](https://llm.datasette.io/en/stable/setup.html#setting-a-custom-directory-location)
+   setting. You can then confirm the path with the following shell command:
+   `dirname "$(llm logs path)"`.
```
```suggestion
   setting. To confirm the path, you can use the following shell command:
   `dirname "$(llm logs path)"`.
```

zerolab commented on docs/ai-backends.md:
````diff
@@ -107,3 +112,47 @@ WAGTAIL_AI = {
+        apikey: your-api-key
+   ```
+3. Set the `MODEL_ID` in the Wagtail AI settings in your Django project
+   settings to the `model_id` you set in the yaml file.
````
```suggestion
   settings file to the `model_id` you added in `extra-openai-models.yaml`.
```

zerolab reviewed (changes requested):
A few minor tweaks and this is good to merge.
Thank you @tm-kn

tm-kn commented:
Thanks for your help @dan!

tm-kn commented:
Thanks for your help @zerolab!
